src/components/organisms/UiMultipleAnswer/UiMultipleAnswer.vue

```diff
@@ -178,6 +185,29 @@ const itemsToRender = computed(() => (props.items.map((item) => {
     },
   };
 })));
+
+const setFirstMultipleAnswerItemRef = (
+  el: Element | ComponentPublicInstance,
+  index: number,
+) => {
+  if (index === 0
+    && 'content' in el
+    && typeof el.content === 'object'
+    && el.content
+    && 'input' in el.content
+    && el.content.input instanceof HTMLInputElement
+  ) firstMultipleAnswerItemRef.value = el.content.input;
+};
+
+watch([
+  hasError,
+  firstMultipleAnswerItemRef,
+], ([
+  errorValue,
+  item,
+]) => {
+  if (errorValue) focusElement(item);
+}, { immediate: true });
```

Comment:
If this watch is based on 2 value `hasError` and `firstMultipleAnswerItemRef` don't need to add `{ immediate: true }`. It will work the same and won't trigger unnecessarily at start.

Reply:
Nice! Thanks

Reply:
But do we really want it to not run on start?